### PR TITLE
Fix ECS task definition for static ingress on fargate

### PR DESCRIPTION
**Pull request by blairboy362**
Hopefully this duplication will be short lived.

### CODE REVIEW

**Comment by blairboy362** [comment]
It's a duplicate of the non-fargate container definition but with the `systemControls` object removed.

**Comment by Krenair** [comment]
looks like you actually removed `linuxParameters`?

**Comment by blairboy362** [comment]
Oh dear. Trust the code, not the comment :) `linuxParameters` was what I intended to remove.

**Review by Krenair — APPROVED** [review]
makes sense from the errors, don't think we should need any special linux permissions for this?

File: terraform/modules/hub/files/tasks/static-ingress-fargate.json [hunk 1]
```diff
@@ -0,0 +1,43 @@
+[
+  {
+    "name": "static-ingress",
+    "image": "${image_identifier}",
+    "cpu": ${allocated_cpu},
+    "memory": ${allocated_memory},
```
Comment (chrisfarms) [inline terraform/modules/hub/files/tasks/static-ingress-fargate.json:6]: non-blocking: these are optional in fargate as the memory is set at the task-level

**Review by chrisfarms — APPROVED** [review]

File: terraform/modules/hub/files/tasks/static-ingress-fargate.json (hunk 1 above)
Comment (blairboy362) [reply terraform/modules/hub/files/tasks/static-ingress-fargate.json:6]: Yeah. It's on my list to tidy these up once the migration to fargate is done.

**Pull request merged**